.github/BOTMETA.yml

```diff
@@ -1388,7 +1388,7 @@ macros:
   team_opennebula: ilicmilan meerkampdvv rsmontero xorel nilsding
   team_oracle: manojmeda mross22 nalsaber
   team_purestorage: bannaych dnix101 genegr lionmax opslounge raekins sdodsley sile16
-  team_redfish: mraineri tomasg2012 xmadsen renxulei rajeevkallur bhavya06 jyundt
+  team_redfish: mraineri tomasg2012 xmadsen renxulei rajeevkallur bhavya06 Gayathirideviramasamy TSKushal
```

Comment:
Please do not remove jyundt.

Reply:
Apologies, must have removed it by mistake. Reverted now. Thanks!